alert-combiner-py/src/findings.py

```diff
@@ -111,18 +111,22 @@ def create_finding(block_chain_indexer, addresses: str, victim_address: str, vic
                        })
 
     @staticmethod
-    def alert_FP(address: str, label: str, metadata: dict) -> Finding:
+    def alert_FP(address: str, label: str, metadata_list: list) -> Finding:
```

Comment:
dont quite understand why we now have a metadata list; i see above you create a list, but only pass in one metadata obj?

Reply:
@christian-forta, there may be more than one labels due to different label metadata. In the code I'm doing

> if entity not in REACTIVE_LIKELY_FPS:
                        REACTIVE_LIKELY_FPS[entity] = {'label': label_name, 'metadata': [metadata]}
                    else:
                        REACTIVE_LIKELY_FPS[entity]['metadata'].append(metadata)
                        
and then passing

`findings.append(AlertCombinerFinding.alert_FP(address, REACTIVE_LIKELY_FPS[address]['label'], REACTIVE_LIKELY_FPS[address]['metadata']))`